### ISSUE TITLE
Allow custom keyboard shortcuts for the click drag motions [enhancement]

### ISSUE THREAD

**Issue by venpopov** [noah-nuebling/Mac-Mouse-Fix#1210]
## Description
Currently the click and drag option for mouse buttons lets you use only two limited actions. It would be great if the other functionality, especially custom keyboard shortcuts were available for the click drag mode as well.

## Rationale
It would give me much more flexibility to perform "gesture-like" custom actions with the mouse. It would increase the number of possible configurations substantially, and let me map intuitive directional commands to the mouse.

## Additional info
–

**Comment by nikitapashinsky** [comment]
Agreed! This and per-app settings would make Mac Mouse Fix the only mouse customization app I need on my Mac.
